scripts/quests/otherAreas/For_the_Birds.lua

```diff
@@ -0,0 +1,152 @@
+-----------------------------------
+-- For the Birds
+-----------------------------------
+-- Log ID: 4, Quest ID: 104
+-- Koblakiq          !pos -64 21 -117
+-- Daa Bola the Seer !pos -157 -17 193
+-- GeFhu Yagudoeye   !pos -91 -3 -127
+-----------------------------------
+require('scripts/globals/items')
+require('scripts/globals/keyitems')
+require('scripts/globals/npc_util')
+require('scripts/globals/quests')
+require('scripts/globals/zone')
+require('scripts/globals/interaction/quest')
+-----------------------------------
+local ID = require('scripts/zones/Beadeaux/IDs')
+-----------------------------------
+local quest = Quest:new(xi.quest.log_id.OTHER_AREAS, xi.quest.id.otherAreas.FOR_THE_BIRDS)
+
+quest.reward =
+{
+    item = xi.items.JAGUAR_MANTLE,
+}
+
+quest.sections =
+{
+    {
+        check = function(player, status, vars)
+            return status == QUEST_AVAILABLE and
+            player:hasCompletedQuest(xi.quest.log_id.OTHER_AREAS, xi.quest.id.otherAreas.MISSIONARY_MOBLIN) and
+            not quest:getMustZone(player)
+        end,
+
+        [xi.zone.OLDTON_MOVALPOLOS] =
+        {
+            ['Koblakiq'] = quest:progressEvent(14, { [1] = xi.items.ARNICA_ROOT }),
+
+            onEventFinish =
+            {
+                [14] = function(player, csid, option, npc)
+                    if option == 1 then
+                        quest:begin(player)
+                    end
+                end,
+            },
+        },
+    },
+    {
+        check = function(player, status, vars)
+            return status == QUEST_ACCEPTED
+        end,
+
+        [xi.zone.OLDTON_MOVALPOLOS] =
+        {
+            ['Koblakiq'] =
+            {
+                onTrigger = function(player, npc)
+                    if quest:getVar(player, 'Prog') == 0 then
```

Comment:
To not hammer on the database fir every condition should local the progress then call the local on the condition if it appears more than once